js/modules/broken-axis.src.js

```diff
@@ -19,6 +19,21 @@ var addEvent = H.addEvent,
     Axis = H.Axis,
     Series = H.Series;
 
+/**
+ * Returns the first break found where the x is larger then break.from and
+ * smaller then break.to.
+ *
+ * @param {number} x The number which should be within a break.
+ * @param {array} breaks The array of breaks to search within.
+ * @return {object|boolean} Returns the first break found that matches, returns
+ * false if no break is found.
+ */
+var findBreakAt = function (x, breaks) {
+    return breaks.find(function (b) {
```

Comment:
Unfortunately `Array.prototype.find` is off limits for now, IE11 doesn't support it. Use `H.find`.

Reply:
Good catch.
I often forget which Array methods that are off limits, is it possible to add an eslint rule or plugin that checks browser compatibility?